de-lib/src/main/java/org/iplantc/de/client/models/tool/ToolAutoBeanFactory.java

```diff
@@ -33,4 +34,20 @@
 
     AutoBean<SimpleServiceError> simpleServiceError();
 
+    /**
+     * Build a tool with default values
+     *
+     * @return  a tool
+     */
+    default AutoBean<Tool> geTool() {
+        Tool tool = getTool().as();
```

Comment:
Will these `getTool` method names conflict?
It seems like this `getTool` method will be the only one accessible from now on?

Which may not be a bad thing, but if we keep this as the only `getTool` method, then we should refactor other places the old method was called from. For example, the call in `ToolAdminDetailsDialog` no longer needs to create and set a default container.